x11-terms/cool-retro-term/cool-retro-term-1.2.0.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg
+
+QTW_PN="qmltermwidget"
+QTW_PV=63228027e1f97c24abb907550b22ee91836929c5
+QTW_P="${QTW_PN}-${QTW_PV}"
+
+DESCRIPTION="Terminal emulator with an old school look and feel"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term/"
+
+SRC_URI="
+	https://github.com/Swordfish90/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/Swordfish90/${QTW_PN}/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
```

Please plain text these parts as much as possible for easier copy-pasting.
```suggestion
	https://github.com/Swordfish90/cool-retro-term/archive/${PV}.tar.gz -> ${P}.tar.gz
	https://github.com/Swordfish90/qmltermwidget/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
```
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI